upgrade/UPGRADE-v11.0.0-dev.md

```diff
@@ -503,6 +503,17 @@ There you can find links to upgrade notes for other versions too.
             +       UploadedFileLocator $uploadedFileLocator
                 )
             ```
+    - `Shopsys\FrontendApiBundle\Controller\FrontendApiController`
+        - property `$graphqlConfigurator` is no longer nullable
```

Comment:
it seems like it has never been nullable (maybe this applies to many and many other properties mentioned here in the upgrade notes in the previous commits, but I think the notes are good enough now, I do not want you to go through them again :smile: )

Reply:
You're right. They were not annotated as nullable, but in some scenarios, they could be null (overridden constructor).

So from my PoV, it seems more like an incorrect annotation before.